src/ray/util/logging.h

```diff
@@ -254,14 +254,23 @@ class RayLog {
   /// This function to judge whether current log is fatal or not.
   bool IsFatal() const;
 
+  /// TODO(hjiang): Having both log directory and filename for `StartRayLog` is confusing,
+  /// at the end of the day should have only filename. Fix all usage in the next PR.
+  ///
   /// The init function of ray log for a program which should be called only once.
   ///
   /// \parem appName The app name which starts the log.
   /// \param severity_threshold Logging threshold for the program.
-  /// \param logDir Logging output file name. If empty, the log won't output to file.
+  /// \param log_dir Logging output directory name.
+  /// \param stdout_log_file Logging output filepath for stdout.
+  ///
+  /// Both [log_dir] and [stdout_log_file] are used to determine log output filename; if
+  /// both empty, the log won't output to file, but to stdout. It's illegal to set
+  /// [stdout_log_file] and [log_dir] at the same time.
   static void StartRayLog(const std::string &appName,
                           RayLogLevel severity_threshold = RayLogLevel::INFO,
-                          const std::string &logDir = "");
+                          const std::string &log_dir = "",
+                          const std::string &stdout_log_filepath = "");
```

Comment:
```suggestion
                          const std::string & log_filepath = "");
```

As we discussed offline, 1stdout log1 is meaningful from python's perspective not from spdlog perspective.

Reply:
Updated.